### PR TITLE
[MOB-4982] adds IterableInAppMessage to index.ts

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-4982](https://iterable.atlassian.net/browse/MOB-4982)

## ✏️ Description

This pull request adds the IterableInAppMessage to the top-level index.ts. Adjusts imports to prevent require cycles.

### CODE REVIEW

File: ts/index.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
 */
 
 import { Iterable } from './Iterable'
+import IterableInAppManager from './IterableInAppManager'
```
Comment (Ayyanchira) [inline ts/index.ts:9]: nit pick: inconsistent with above line. {} present in line 8 while its not used in line 9

File: ts/IterableInboxMessageDisplay.tsx [hunk 2]
```diff
@@ -6,14 +6,14 @@ import {
    View,
    ScrollView,
    StyleSheet,
-   Platform,
```
Comment (Ayyanchira) [inline ts/IterableInboxMessageDisplay.tsx:9]: Why is platform removed?

**Review by Ayyanchira — COMMENTED** [review]
Added some comments. 
Also, this might be a breaking change for devs importing new SDK right? Might have to document that as well?

File: ts/index.ts (hunk 1 above)
Comment (evangreer91) [reply ts/index.ts:9]: IterableInAppManager is a default export, while Iterable is a named export. see [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export).

File: ts/IterableInboxMessageDisplay.tsx (hunk 2 above)
Comment (evangreer91) [reply ts/IterableInboxMessageDisplay.tsx:9]: Platform is not used in the component.

**Review by Ayyanchira — APPROVED** [review]

**Pull request merged**